playground/package.json

```diff
@@ -1,59 +1,46 @@
 {
 	"name": "@markuplint/playground",
 	"description": "https://playground.markuplint.dev",
-	"version": "2.1.0-alpha.13",
 	"private": true,
+	"version": "0.0.0",
+	"type": "module",
 	"scripts": {
-		"pg:dev": "yarn; webpack-dev-server --progress",
-		"pg:build": "yarn; webpack --mode production"
-	},
-	"author": "Yusuke Hirao",
-	"license": "MIT",
-	"devDependencies": {
-		"@types/mini-css-extract-plugin": "2.5.1",
-		"@types/script-ext-html-webpack-plugin": "2.1.3",
-		"@types/webpack-dev-server": "4.7.2",
-		"autoprefixer": "10.4.14",
-		"babel-loader": "9.1.2",
-		"css-loader": "6.7.3",
-		"cssnano": "6.0.1",
-		"file-loader": "6.2.0",
-		"html-inline-css-webpack-plugin": "1.11.1",
-		"html-loader": "4.2.0",
-		"html-webpack-plugin": "5.5.1",
-		"json-loader": "0.5.7",
-		"mini-css-extract-plugin": "2.7.5",
-		"monaco-editor-webpack-plugin": "7.0.1",
-		"postcss-base64": "0.7.1",
-		"postcss-calc": "9.0.0",
-		"postcss-custom-media": "9.1.3",
-		"postcss-gap-properties": "4.0.1",
-		"postcss-loader": "7.3.0",
-		"postcss-math": "0.0.10",
-		"sass": "1.62.1",
-		"sass-loader": "13.2.2",
-		"script-ext-html-webpack-plugin": "2.1.5",
-		"style-loader": "3.3.2",
-		"svelte-loader": "3.1.7",
-		"svelte-preprocess": "5.0.3",
-		"ts-loader": "9.4.2",
-		"ttf-loader": "1.0.2",
-		"webpack": "5.82.0",
-		"webpack-cli": "5.0.2",
-		"webpack-dev-server": "4.13.3"
+		"pg:dev": "vite",
+		"pg:build": "tsc && vite build",
+		"pg:lint": "eslint src --ext ts,tsx",
+		"pg:preview": "vite preview"
 	},
 	"dependencies": {
-		"@markuplint/html-parser": "3.8.0",
-		"@markuplint/html-spec": "3.9.0",
-		"@markuplint/i18n": "3.8.0",
-		"@markuplint/ml-ast": "3.1.0",
-		"@markuplint/ml-config": "3.9.0",
-		"@markuplint/ml-core": "3.10.0",
-		"@markuplint/ml-spec": "3.9.0",
-		"@markuplint/parser-utils": "3.8.0",
-		"@markuplint/rules": "3.10.0",
-		"monaco-editor": "0.38.0",
-		"svelte": "3.58.0",
-		"tslib": "2.5.0"
+		"@headlessui/react": "^1.7.14",
+		"@markuplint/parser-utils": "^3.7.0",
+		"@monaco-editor/react": "^4.5.0",
+		"@webcontainer/api": "^1.1.3",
+		"ansi-regex": "^6.0.1",
+		"markuplint": "^3.8.0",
```

Comment:
@kagankan 

Could you please explain the relationship between the packages needed here and those referenced in `deps.json`?

https://github.com/markuplint/markuplint/pull/973/files#diff-63fccff55aced03b2348fd7c4970b2d86a885b8ed64cec1cd81c686ae7e0d7d2

Reply:
@YusukeHirao 


`playground/package.json` defines packages installed during development and build.
On the other hand, `deps.json` defines packages installed on the browser (via WebContainer API). The `markuplint` defined in this `deps.json` is executed as a linter on the playground site.

Only type information (`MLResultInfo`) is used from `markuplint` installed in `playground/package.json`.
https://github.com/markuplint/markuplint/pull/973/files#diff-c34e3203108664d9deef6eb19163ebbc58121db73b79dc4831784369bf3dcc03

So, if there is a change in the type information, a runtime error may occur, but I'm assuming no change in that.
(Please let me know if there is an appropriate way to handle this)

Reply:
@YusukeHirao 

Do you have any additional questions?
Please let me know if my explanation is not clear enough.